src/tests/jobtest.c

```diff
@@ -30,26 +30,131 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
 #include <daos.h>
+#include <daos_mgmt.h>
+#include <daos/common.h>
+#include <daos/pool.h>
+
+static char *progname;
+
+void print_usage(void)
+{
+	fprintf(stderr, "Usage: %s -p pool_str [-s nsecs] [-x] "
+			"[-h handles_per_pool]\n", progname);
+}
+
+void cleanup_handles(uuid_t *pool_uuids, int num_pools,
+		     daos_handle_t **pool_handles, int handles_per_pool)
+{
+	int i, rc;
+
+	for (i = 0; i < num_pools; i++) {
+		int idx;
+
+		if (uuid_is_null(pool_uuids[i])) {
+			continue;
+		}
+		for (idx = 0; idx < handles_per_pool; idx++) {
+			if (daos_handle_is_inval(pool_handles[i][idx])) {
+				continue;
+			}
+			rc = daos_pool_disconnect(pool_handles[i][idx], NULL);
+			if (rc) {
+				struct dc_pool	*pool;
+				char		pool_str[64];
+				char		hdl_str[64];
+
+				pool = dc_hdl2pool(pool_handles[i][idx]);
+				uuid_unparse_lower(pool->dp_pool, pool_str);
+				uuid_unparse_lower(pool->dp_pool, hdl_str);
+				printf("disconnect handle %s from pool %s "
+					"failed\n", hdl_str, pool_str);
+			}
+		}
+		free(pool_handles[i]);
+		pool_handles[i] = NULL;
+	}
+}
+
+/*
+ * Takes in a comma separated list of UUIDs and turns them into a list of
+ * UUID structures. It will return the number of entries on success and -1 on
+ * failure
+ */
+int parse_pool_handles(char *hndl_str, uuid_t **handles)
+{
+	char		*idx, *pool_str;
+	const char	delim[2] = ",";
+	int		hdl_count = 1 /* we have at least 1 */, i = 0, rc;
```

Comment:
We're not exactly consistent about this, but the coding style guide says we should define one variable per line, even if we have several of the same type. I think in cases with value assignments, like here, we should definitely follow that rule.

Reply:
I am not sure if you need to or not, but I noticed that some of the code uses the Doxygen format and others do not.  I think we are striving for all of the C code to use it though.  So perhaps you should be converting your new comments to that format, too.

Reply:
I was under the impression that doxygen comments were for things that we need to generate API documentation for? Also fixed Kris's comment.

Reply:
Yeah, the Doxygen comments are turned into documentation. I don't think we'd want this included.